src/helpers/isTokenUserOnline.js

```diff
@@ -0,0 +1,40 @@
+import {api} from 'helpers/api';
+
+const isDefined = (value) => {
+    return value !== undefined && value !== null;
+};
+
+async function callAPI(token, id) {
+    await api.get(`/users/${id}`).then(res => {
+
+        // check if user is online and matches the token
+        if ((res.data.token === token) && (res.data.status === "ONLINE")) {
```

Comment:
should probably be if (res.headers.token === token ) && (res.data.status === "online") or, what I would prefer, => return res.data (=== true) [MUST]